README.md

```diff
@@ -1,15 +1,51 @@
-# mollymawk
+# Mollymawk
 
 From Wikipedia: The mollymawks are a group of medium-sized albatrosses that form the genus Thalassarche.
 
 A web interface for remotely managing [albatross](https://github.com/robur-coop/albatross).
 
 Both as an interactive user interface to point and click deploying and destroying unikernels, as well as a REST API with authentication.
 
-Mollymawk is a unikernel itself, and communicates via TLS with albatross. It preserves state about users and configuration for the remote albatross instance.
+Mollymawk is a unikernel itself, and communicates via TLS with albatross. It preserves state about users and configuration for the remote albatross instance(s).
 
 The assumption is that only mollymawk maintains and modifies the albatross that is running - it is not supported that you run other administrative tasks (such as modifying the policies) while mollymawk is running.
 
+## Using Mollymawk with multiple Albatross installations
+
+With **Mollymawk**, you can now manage **multiple Albatross** instances.
+
+---
+
+### Information needed per instance
+
+* **Name** – Friendly label (e.g., `prod-eu-1`)
+* **Host / IP** – e.g., `albatross.example.org` or `10.0.42.15`
```

Comment:
when looking in configuration.ml the of_json part, there's the line "let server_ip = Ipaddr.of_string <user-data>" -- so I wonder how you'd enter a hostname and where/how that would be resolved?